server/config.ts

```diff
@@ -77,6 +77,14 @@ export default {
       },
       agent: new AgentConfig(Number(get('NOMIS_PRISONER_TIMEOUT_RESPONSE', 60000))),
     },
+    activities: {
```

Comment:
do we also need to change the healthcheck for this service  with the new dependency?

Reply:
Yes!